lib/unifex/interface_IO.ex

```diff
@@ -20,8 +20,9 @@ defmodule Unifex.InterfaceIO do
   end
 
   def store_interface!(name, dir, code) do
-    {header, source} = code
-    out_dir_name = Path.join(dir, @generated_dir_name)
+    {header, source, interface} = code
+    interface = Unifex.Helper.get_module_string(interface) |> String.downcase()
```

Comment:
I am not sure how we want to store generated files. At this moment we have e.g.
* CNode - interface name
* BUNDLEX_CNODE - compiler flag
* cnode - directory in which generated files for this interface are stored.

Maybe we should change directory name for the same as interface (generator module name) but this doesn't look too good.